default.hbs

```diff
@@ -23,16 +23,16 @@
     {{ghost_foot}}
 
     <script src={{asset "built/js/vendor/swiper/swiper.js"}}></script>
-    <script src={{asset "built/js/vendor/big_picture/BigPicture.js"}}></script>
     <script src={{asset "built/js/vendor/jarallax/jarallax.js"}}></script>
-    <script src={{asset "built/js/vendor/isotope/isotope.js"}}></script>
     <script src={{asset "built/js/vendor/images_loaded/imagesloaded.pkgd.min.js"}}></script>
     <script src={{asset "built/js/vendor/easytimer/easytimer.js"}}></script>
-    <script src={{asset "built/js/vendor/pie_chart/pie_chart.js"}}></script>
     <script src={{asset "built/js/vendor/ghost_finder/ghost_finder.js"}}></script>
+    <script async src={{asset "built/js/vendor/big_picture/BigPicture.js"}}></script>
+    <script async src={{asset "built/js/vendor/isotope/isotope.js"}}></script>
+    <script async src={{asset "built/js/vendor/pie_chart/pie_chart.js"}}></script>
 
     <script src={{asset "built/js/bootstrap.js"}}></script>
-    <script src={{asset "built/js/theme.js"}}></script>
-    <script src={{asset "built/js/vendor/pure_counter/purecounter.js"}}></script>
+    <script async src={{asset "built/js/theme.js"}}></script>
```

Comment:
This doesn't work.  There is no guarantee that theme.js will be loaded after all the other js files that it depends upon.  We can't do async for js files that are configured with theme.js.  We can do defer.  Deferred js files are guaranteed to be loaded in order.